pkg/operator/sync.go

```diff
@@ -1901,3 +1902,38 @@ func cmToData(cm *corev1.ConfigMap, key string) ([]byte, error) {
 	}
 	return nil, fmt.Errorf("%s not found in %s/%s", key, cm.Namespace, cm.Name)
 }
+
+// Validates configuration provided in the MachineConfiguration object's spec for each feature
+// and updates the status of the object as necessary
+func (optr *Operator) syncMachineConfiguration(_ *renderConfig) error {
+
+	// Grab the cluster CR
+	mcop, err := optr.mcopLister.Get("cluster")
+	// Create one if it doesn't exist
+	if mcop == nil || apierrors.IsNotFound(err) {
```

Comment:
are you checking if mcop is nil as an alternative to 
```
if err != nil {
    if  apierrors.IsNotFound(err) {
        // handle not found
     }
     // handle regular error
}
```

Reply:
im not entirely sure why I went that direction, I think yours makes more sense - I'll update to that pattern in my next pass so I can handle normal errors too

Reply:
Fixed - I think when I originally wrote this, I was getting lint errors as `mcop` wasn't being used, I must've done this to get around that during testing and forgot to clean it up. Thanks!